release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1269,6 +1269,27 @@ Optionally, you can now override the default evaluation intervals for all polici
 
 For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-configuring-pgt-compliance-eval-timeouts_ztp-advanced-policy-config[Configuring policy compliance evaluation timeouts for PolicyGenTemplate CRs].
 
+[id="ocp-4-12-0-ironic-agent-image-ztp"]
+==== Ironic agent mirroring in disconnected GitOps ZTP installations
+
+For disconnected installations using GitOps ZTP, if you are deploying {product-title} version 4.11 or earlier to a spoke cluster with converged flow enabled, you must mirror the default Ironic agent image to the local image repository. The default Ironic agent images are the following:
+
+* AMD64 Ironic agent image: `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:d3f1d4d3cd5fbcf1b9249dd71d01be4b901d337fdc5f8f66569eb71df4d9d446`
+
+* AArch64 Ironic agent image: `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:cb0edf19fffc17f542a7efae76939b1e9757dc75782d4727fb0aa77ed5809b43`
+
+For more information about mirroring images, see xref:../installing/disconnected_install/installing-mirroring-installation-images.adoc#installation-mirror-repository_installing-mirroring-installation-images[Mirroring the OpenShift Container Platform image repository]
+
+[id="support-for-dual-stack-addressing-for-the-API-VIP-and-Ingress-VIP"]
```

Comment:
Can we explicitly mention that in 4.12 dual stack is supported on baremetal only? We've gotten a few questions about that recently which seem to be related to a release note about the dual stack VIP feature. We added the ability to configure multiple VIPs for all platforms, but it's still only supported on baremetal for now.

Reply:
Updated as suggested.